src/components/tables/FirsColumn.tsx

```diff
@@ -0,0 +1,21 @@
+import { Box } from '@mui/system'
+import { Link } from 'react-router-dom'
+
+interface IFirstColumn {
+  image: string
+  value: string
+  path: string
+}
+
+const FirsColumn = (props: IFirstColumn) => {
```

Comment:
The component should have a more meaningful name. It can't be used for any first column, it's specifically for a coin. `CoinColumnHeader` or something like that would make more sense.